Kill.py

```diff
@@ -0,0 +1,71 @@
+""" Kill Process Extension """
+
+import os
+import getpass
+from albertv0 import *
+from subprocess import Popen, PIPE
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Kill Process"
+__version__ = "1.0"
+__trigger__ = "kill"
```

Comment:
Please use space after triggers, otherwise "Killing Hasselhoff" would trigger this extension